ui/templates/ui/help.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block title %}OVS | Video Detail{% endblock %}
+{% block pagetitle %}OVS: Video Detail{% endblock %}
```

Comment:
Missed this during the first review. title & pagetitle block text needs to be updated

Reply:
lol :woman_facepalming: